test_regress/vpi_main.cpp

```diff
@@ -0,0 +1,180 @@
+// Copyright cocotb contributors
+// Licensed under the Revised BSD License, see LICENSE for details.
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Rename e.g. t_vpi_main.cpp.

Is this really from cocotb?

Reply:
Yes, but it looks like they also copied the main from verilator via make_main, so I'll change it back to the verilator doc header